docs/tutorials/Hosting_on_Heroku.md

```diff
@@ -0,0 +1,197 @@
+
+Once your app is installed copy the **Guild ID** for the server where you installed your app.
+
+## Setting up Heroku
+
+The first thing you should do is set up an account for Heroku.
+
+Head over to https://id.heroku.com/login and log in to your account, or [set up a new account](https://signup.heroku.com/) if you don’t have one.
+
+![Heroku login](heroku-login.png)
+
+After logging in, at the top right corner of your account dashboard, click New > Create new app. You can then name it whatever you want.
+
+### Adding a Procfile
+
+For your app to work with Heroku, you'll need to add a `Procfile`. **Create a new file called `Procfile`** where you will add a short script that Heroku will run when starting up your app.
+
+> INFO
+> For folks following the tutorial and using the [sample repo](https://github.com/discord/heroku-sample-app), the `Procfile` will already be configured appropriately. Otherwise, the script in your `Procfile` will vary depending on what language you're coding in.
+
+```bash
+# Procfile contents for bakerbot (startup)
+worker: node index.js
+```
+
+You now have an empty application for Heroku that's ready to be linked to your repository.
+
+## Connecting Heroku to GitHub
+
+In this tutorial, we'll use a GitHub repository as the source for the Heroku app. Before you can deploy, you should [push your project to GitHub](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-new-repository).
+
+> INFO
+> If you prefer to deploy from your local app, you can [use the Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli)
+
+After you have your GitHub repo and Heroku application set up, you will be met with some options in the **Deploy** tab. From here, you have several deployment method options to choose from
+
+![Heroku dashboard](heroku-connectGH.png)
+
+Click **Connect this app to GitHub**.
+ 
+> INFO
+> that you can only connect Heroku apps to a single GitHub repository. 
+
+From there, you will be able to manage the app's deployment method:
+
+![Heroku deployment](heroku-deploy.png)
+
+Selecting a branch and enabling **Automatic Deploys** for your app will deploy a new version of this app every time a push occurs. **Manual Deploys** would require you to select a specific branch before updates go live for your bot. 
+
+> INFO
+> If you want to do this with the Heroku CLI, there are [separate steps in the dashboard](https://devcenter.heroku.com/articles/heroku-cli) you can follow to accomplish this.
+
+## Configuring app in Heroku
+
+Before your app can go online, you'll have to configure your Heroku environment with your Discord bot's credentials.
+
+### Adding app credentials
+
+In the **Settings** tab, add your bot’s `TOKEN`, `GUILD_ID`, `CLIENT_ID` from [the section above](#creating-your-discord-app), and any other tokens or API keys you want for your own app.
```

Comment:
Since you can get a preview of this by building locally, does this actually work as expected? It makes intuitive sense to do this, and it'd make a lot of links work in the GH render of the docs, but IIRC it doesn't work on the deployed site in its current state.

Could we get it to work on the deployed site?

If it doesn't work here's the fix:

```suggestion
In the **Settings** tab, add your bot’s `TOKEN`, `GUILD_ID`, `CLIENT_ID` from [the section above](#DOCS_TUTORIALS_HOSTING_ON_HEROKU/creating-your-discord-app), and any other tokens or API keys you want for your own app.
```

Reply:
yah i think you're right—i don't think it'll resolve the current one.